### DISCUSSION TITLE
Is there a prisma version of this boilerplate?

### DISCUSSION THREAD

**Discussion by khantseithu** [brocoders/nestjs-boilerplate#1230] [Ideas]
I need to quickly bootstrap a new nest js project, which will be using prisma as ORM. What are your suggestions on "to integrate prisma with this boilerplate" ?

**Comment by Shchepotin** [comment]
Currently, we have no plans to use Prisma in the NestJS Boilerplate.

### Why?

We don't use Prisma in our projects because it does many redundant queries, making many unnecessary loads on a database. Example: https://github.com/prisma/prisma/issues/21447
